improve_C1 branch Game screen improved version 확인 요망

1000peach opened the pull request:
v0.2.0 Game screen improved version 변경 사항

console size 조절
- printBoard 함수의 내용 구현: 잘못된 출력 제거, 함수화를 통해 main.c 에서의 중복 제거
- textcolor 함수를 통해 숫자 별로 색 부여
ㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡ pull request 이후 수정 사항
- 중괄호 맞지 않는 부분 수정
- printBoard, printScore 함수 분리
- gameboard menu, 색상값을 define

전체적인 branch의 개선 목적: 게임화면을 시각적으로 편리하게 개선
총 3 commits의 게임 화면 개선 버전 improve_C1 입니다.
pull request에서 요구받은 수정 사항들은 개선하였습니다.

더 자세한 변경 사항은 commit message에 적어두었습니다.
확인하시고 수정사항 또는 오류가 발생하면 코멘트로 남겨주세요!
@LeeEunhyung @wlals7217

LeeEunhyung commented:
콘솔창 조절하는 코드가 Linux 환경에서 먹히지 않아 다른 방식의 함수를 설정해야할 것같아요! 해당 부분은 제가 improve_L2 branch에서 개선하도록 하겠습니다! 수고하셨습니다!!

wlals7217 commented:
확인했습니다! 수고하셨습니다. 머지 찬성합니다!

1000peach commented:
동의 받았으니 merge 하겠습니다!

1000peach committed:
Change console size

- console size를 게임 화면에 맞게 조절해 원활한 게임 환경 개선

1000peach committed:
Make function (printBoard)

- printBoard: gameboard, 상태 출력 부분 함수화-> 잘못된 출력 제거, 간단히 구현

1000peach committed:
Improve game screen

- printBoard 개선: 키 안내 출력을 한 눈에 들어오게 수정,
  gameboard에서 숫자 별로 색 구분
- textcolor 함수 추가: 숫자에 색 부여하는 기능
- 전체적으로 시각적으로 편리한 game screen 개선

1000peach committed:
Modify requirements

- printBoard, printScore 함수 분리
- 여는 중괄호의 위치를 규칙에 맞게 수정
- gameboard menu, 색상값을 define

1000peach commented:
v0.2.0 Game screen improved version
